lib/api/src/lib.rs

```diff
@@ -81,7 +81,17 @@ pub use wasmer_types::{
     Atomically, Bytes, GlobalInit, LocalFunctionIndex, MemoryView, Pages, ValueType,
     WASM_MAX_PAGES, WASM_MIN_PAGES, WASM_PAGE_SIZE,
 };
+
+// TODO: should those be moved into wasmer::vm as well?
```

Comment:
Perhaps, but they should probably stay at the top level too!

Reply:
okay, feel free to just kick out the comment when you are happy with the setup